activities.json

```
@@ -602,6 +602,18 @@
     "title": "Picture-in-Picture",
     "url": "https://wicg.github.io/picture-in-picture/"
   },
+  {
+    "ciuName": null,
+    "description": "This specification defines a mechanism that allows for rendering of, and seamless navigation to, embedded content.",
+    "id": "portals",
+    "mozBugUrl": null,
+    "mozPosition": "harmful",
+    "mozPositionDetail": "The Portals proposal introduces a significant amount of complexity to the web without either (a) clearly explaining how it differs and doesn't differ from the iframe element or (b) a serious attempt to specify how it interacts with the large number of existing web features (including aspects of the origin model) many of which are critical for security and how it interacts with existing privacy features in web browsers (such as those that depend on partitioning of storage based on origins).  We believe the proposal as it exists today is harmful, but would be willing to reconsider given a more clearly motivated and clearly specified proposal.",
+    "mozPositionIssue": 157,
+    "org": "Proposal",
+    "title": "Portals",
+    "url": "https://wicg.github.io/portals"
```

```suggestion
    "url": "https://wicg.github.io/portals/"
```
I think we decided to add slashes here to avoid redirects?